com.unity.template-hd/ProjectSettings/QualitySettings.asset

```diff
@@ -115,7 +115,10 @@ QualitySettings:
     customRenderPipeline: {fileID: 11400000, guid: 168a2336534e4e043b2a210b6f8d379a, type: 2}
     excludedTargetPlatforms: []
   m_PerPlatformDefaultQuality:
+    GameCoreScarlett: 1
+    GameCoreXboxOne: 1
     PS4: 1
+    PS5: 1
```

Comment:
Any reason it is set to 1 instead of 5 (ultra)?

Reply:
This QualitySettings only has 3 settings, High Quality (0), Medium Quality (1) and Low Quality (2). PS4 is set to Medium Quality so for the sake of parity PS5 is set to Medium Quality.